Craig_Campbell/lib/greet.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var greeting = exports.greet = function greet(time, name) {
```

Comment:
If you're going to save `exports` to a variable, put `module.exports` in there too to keep them pointing at the same thing (I am still not 100% on this, so if you think you have it correct, let's go see Tyler and make sure!)